integration-cli/docker_cli_run_test.go

```diff
@@ -1958,12 +2127,25 @@ func (s *DockerSuite) TestRunExposePort(c *check.C) {
 }
 
 func (s *DockerSuite) TestRunUnknownCommand(c *check.C) {
-	testRequires(c, NativeExecDriver, DaemonIsLinux)
+	if daemonPlatform != "windows" {
+		testRequires(c, NativeExecDriver)
+	}
 	out, _, _ := dockerCmdWithStdoutStderr(c, "create", "busybox", "/bin/nada")
 
 	cID := strings.TrimSpace(out)
 	_, _, err := dockerCmdWithError("start", cID)
-	c.Assert(err, check.NotNil)
+
+	// Windows and Linux are different here by architectural design. Linux will
+	// fail to start the container, so an error is expected. Windows will
+	// successfully start the container, and once started attempt to execute
+	// the command which will fail.
+	if daemonPlatform == "windows" {
+		// Wait for it to exit.
+		waitExited(cID, 30)
```

Comment:
On SSD, server core can take up to 6 seconds. On spinning, that will be more (circa 10 I suspect, but I only have SSDs ;) ). I was trying to ensure the test was reliable. If it takes 30 seconds, that should be indicative of a failure - I wanted to set something high enough that it -should- never reach.